site/docs/id/plugins/keyboard.md

```diff
@@ -54,43 +50,29 @@ const inlineKeyboard = new InlineKeyboard()
   .text("31 »", "akhir");
 ```
 
-##### Hasil
-
-![Hasil contoh 1](/images/inline-keyboard-example-1.webp)
-
-#### Contoh 2
-
-Tombol berbagi untuk keyboard inline bisa dibuat dengan cara seperti ini:
-
-##### Kode
-
-```ts
-const inlineKeyboard = new InlineKeyboard()
-  .text("Get random music", "acak").row()
-  .switchInline("Send music to friends");
-```
-
-##### Hasil
-
-![Hasil contoh 2](/images/inline-keyboard-example-2.webp)
+Sebuah baris tombol baru bisa ditambahkan dengan cara memanggil `.row()`.
+Kamu juga bisa menggunakan berbagai method yang lain, misalnya `.url()` yang berfungsi agar perangkat pengguna mampu membuka sebuah link tertentu atau hal-hal keren lainnya.
+Silahkan lihat [daftar semua method](https://deno.land/x/grammy/mod.ts?s=InlineKeyboard#Methods) yang tersedia di class `InlineKeyboard`.
 
-#### Contoh 3
+Jika kamu sudah memiliki sebuah array string yang hendak diubah menjadi sebuah keyboard inline, kamu bisa menggunakan gaya penulisan yang kedua.
+Class `InlineKeyboard` memiliki beberapa method static, salah satunya `InlineKeyboard.text` yang berfungsi untuk membuat beberapa object tombol.
+Setelah itu, kamu bisa membuat sebuah instance keyboard inline dari array object tombol tadi menggunakan `InlineKeyboard.from`.
 
-Tombol URL bisa dibuat dengan cara seperti ini:
-
-##### Kode
+Dengan begitu, kamu bisa membuat keyboard inline di atas dengan gaya penulisan fungsional.
 
 ```ts
-const inlineKeyboard = new InlineKeyboard().url(
-  "Read on TechCrunch",
-  "https://techcrunch.com/2016/04/11/this-is-the-htc-10/",
-);
+const labelDataPairs = [
+  ["« 1", "awal"],
+  ["‹ 3", "sebelumnya"],
+  ["· 4 ·", "saat ini"],
+  ["5 ›", "selanjutnya"],
+  ["31 »", "akhir"],
```

Comment:
@CikiMomogi @fadzikri is it intentional that you have translated the callback query data on this page?

Reply:
It is intentional. Yes.

Reply:
Alright